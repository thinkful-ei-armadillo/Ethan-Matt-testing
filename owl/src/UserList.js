import React from 'react';
import User from './User';

function UserList (props) {

  const UserList = props.users.map((u) => {

    return (<User
      id={u.id}
      name={u.name}
      avatar={u.avatar}
      inSession={u.inSession}
      onStage={u.onStage}
      key={u.id}
    /> )
  });

  return (
    <div>
      {UserList}
    </div>
  );
}

export default UserList;
