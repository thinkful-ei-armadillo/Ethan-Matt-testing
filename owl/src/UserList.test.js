import React from 'react';
import ReactDOM from 'react-dom';
import UserList from './UserList';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<UserList />, div);
  ReactDOM.unmountComponentAtNode(div);
});
