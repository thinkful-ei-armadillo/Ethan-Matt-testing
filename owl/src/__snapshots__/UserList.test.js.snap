// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div>
  <div
    className="user"
  >
    <img
      alt="Koren Templeton"
      className="user-avatar"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <div>
      <h6
        className="user-name"
      >
        Koren Templeton
      </h6>
      <div
        className="user-status"
      >
        on stage
      </div>
    </div>
  </div>
  <div
    className="user"
  >
    <img
      alt="Caty Flucker"
      className="user-avatar"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <div>
      <h6
        className="user-name"
      >
        Caty Flucker
      </h6>
      <div
        className="user-status"
      >
        
      </div>
    </div>
  </div>
  <div
    className="user"
  >
    <img
      alt="Axe Kubicka"
      className="user-avatar"
      src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
    />
    <div>
      <h6
        className="user-name"
      >
        Axe Kubicka
      </h6>
      <div
        className="user-status"
      >
        
      </div>
    </div>
  </div>
  <div
    className="user"
  >
    <img
      alt="Frank Runciman"
      className="user-avatar"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <div>
      <h6
        className="user-name"
      >
        Frank Runciman
      </h6>
      <div
        className="user-status"
      >
        
      </div>
    </div>
  </div>
  <div
    className="user"
  >
    <img
      alt="Ashla Attwool"
      className="user-avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <div>
      <h6
        className="user-name"
      >
        Ashla Attwool
      </h6>
      <div
        className="user-status"
      >
        on stage
      </div>
    </div>
  </div>
</div>
`;
